<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUpDialog</class>
 <widget class="QDialog" name="SignUpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>357</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>393</width>
    <height>357</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>393</width>
    <height>357</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sign Up</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>66</width>
     <height>14</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>&lt;span style=&quot; color:#ff0000;&quot;&gt;* &lt;/span&gt;Username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>66</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;span style=&quot; color:#ff0000;&quot;&gt;* &lt;/span&gt;Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>114</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;span style=&quot; color:#ff0000;&quot;&gt;* &lt;/span&gt;Confirm Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>48</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;span style=&quot; color:#ff0000;&quot;&gt;* &lt;/span&gt;Email:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>84</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;span style=&quot; color:#ff0000;&quot;&gt;* &lt;/span&gt;Recommender:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineUsername">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePassword">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineConfirmPassword">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEmail">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineRecommender">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSubmit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Submit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
